app/src/main/AndroidManifest.xml

```diff
@@ -52,13 +50,12 @@
                 <action android:name="android.intent.action.MAIN" />
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
-        </activity>
-<!--        <activity-->
-<!--            android:name=".ChatListActivity"-->
-<!--            android:exported="true" />-->
-<!--        <activity-->
-<!--            android:name=".ProfileActivity"-->
-<!--            android:exported="true" />-->
+        </activity> <!-- <activity -->
+        <!-- android:name=".ChatListActivity" -->
+        <!-- android:exported="true" /> -->
+        <!-- <activity -->
+        <!-- android:name=".ProfileActivity" -->
+        <!-- android:exported="true" /> -->
```

Comment:
Hey Artur, could you remove those commented codes since they are not activity now

Reply:
Done!